### PR TITLE
[Lottie Exporter] Add support for aTan2 convert method

### PR DESCRIPTION
**Pull request by saurabh-ramola**
Added [aTan2](https://wiki.synfig.org/Convert#aTan2) convert method for Lottie Exporter plugin.
@AnishGG , @ice0 , please review :smiley: 

Note: aTan2 convert method works with angle-valued parameters.

### CODE REVIEW

**Comment by saurabh-ramola** [comment]
Added support for [Vector Angle](https://wiki.synfig.org/Convert#Vector_Angle) convert method for Lottie Exporter plugin. This convert method is similar to aTan2 convert method.

@AnishGG , @ice0 , Please review.

File: synfig-studio/plugins/lottie-exporter/common/Param.py [hunk 1]
```diff
@@ -518,6 +518,27 @@ def recur_animate(self, anim_type):
...
                 return ret, self.expression_controllers
 
+            elif self.param[0].tag == "atan2":
+                self.subparams["atan2"].extract_subparams()
+                y, eff_1 = self.subparams["atan2"].subparams["y"].recur_animate(anim_type)
+                x, eff_2 = self.subparams["atan2"].subparams["x"].recur_animate(anim_type)
+                self.expression_controllers.extend(eff_1)
+                self.expression_controllers.extend(eff_2)
+                
+                ret = "div(Math.atan2({y},{x}),degreesToRadians)"
```
Comment (AnishGG) [inline synfig-studio/plugins/lottie-exporter/common/Param.py:528]: This expression does not seem right to me, Hence I tested changing the angle of a group layer, and it does not change anything. I guess you need to use something like:
`ret = "radiansToDegrees(Math.atan2({y}, {x}))"`
You might need to modify the above expression accordingly.

Comment (AnishGG) [inline synfig-studio/plugins/lottie-exporter/common/Param.py:528]: This expression does not seem right to me, Hence I tested changing the angle of a group layer, and it does not change anything. I guess you need to use something like:
`ret = "radiansToDegrees(Math.atan2({y}, {x}))"`
You might need to modify the above expression accordingly.

Comment (saurabh-ramola) [reply synfig-studio/plugins/lottie-exporter/common/Param.py]: Thank you for pointing out the mistake, I have corrected it now.

Comment (AnishGG) [reply synfig-studio/plugins/lottie-exporter/common/Param.py]: Please make a seperate P.R. for vectorangle convert method. I will review this expression shortly. ^__^

Comment (saurabh-ramola) [reply synfig-studio/plugins/lottie-exporter/common/Param.py]: Done

File: synfig-studio/plugins/lottie-exporter/common/Param.py [hunk 2]
```diff
@@ -518,6 +518,18 @@ def recur_animate(self, anim_type):
...
                 return ret, self.expression_controllers
 
+            elif self.param[0].tag == "atan2":
+                self.subparams["atan2"].extract_subparams()
+                y, eff_1 = self.subparams["atan2"].subparams["y"].recur_animate(anim_type)
+                x, eff_2 = self.subparams["atan2"].subparams["x"].recur_animate(anim_type)
+                self.expression_controllers.extend(eff_1)
+                self.expression_controllers.extend(eff_2)
+                
+                ret = "radiansToDegrees(Math.atan2({y}, {x}))"
```
Comment (AnishGG) [inline synfig-studio/plugins/lottie-exporter/common/Param.py:528]: This expression is not giving the correct angle. You might need to add/subtract 180 degrees depending upon the axis used by lottie. Please try this expression on some more examples.

Comment (saurabh-ramola) [reply synfig-studio/plugins/lottie-exporter/common/Param.py]: Added a fix for that mistake, works fine depending on the axis now ^_^

Comment (AnishGG) [reply synfig-studio/plugins/lottie-exporter/common/Param.py]: Yep, works perfectly now!
Nice work.

**Comment by AnishGG** [comment]
@ice0 I guess this PR is ready for merge. ^__^

**Comment by ice0** [comment]
Merged. Thank you!

**Pull request merged**